src/Provider/Provider.Web/Views/Shared/_LayoutWithoutNavigation.cshtml

```diff
@@ -150,7 +136,7 @@ public string GetTitle()
         </div>
     </footer>
     <script src="/lib/govuk-frontend/dist/govuk-frontend-2.5.0.min.js"></script>
-    <script src="/lib/esfa-stacker/stacker.js"></script>
+    <script src="/lib/esfa-provider-menu/navigation.js"></script>
```

Comment:
I think we should add `asp-append-version="true"` to aid cache busting.  Add it to the link above too.

Reply:
Have added the attribute for the navigation.js.